controllers/mysqlcluster_controller_test.go

```diff
@@ -1612,4 +1613,264 @@ var _ = Describe("MySQLCluster reconciler", func() {
 			return nil
 		}).Should(Succeed())
 	})
+
+	It("should sets ConditionUpToDate to be true when StatefulSet is ready", func() {
+		cluster := testNewMySQLCluster("test")
+		err := k8sClient.Create(ctx, cluster)
+		Expect(err).NotTo(HaveOccurred())
+
```

Comment:
Please add a test when the sts.ststus is empty.

Reply:
I added a test that checks condition is false when status is empty.